README.md

```diff
@@ -17,13 +17,13 @@ You can use inuitcss in your project by installing it using a package manager
 **npm:**
 
 ```
-$ npm install --save-dev inuitcss
```

Comment:
Why would we not save this as dependency in `package.json` though? I think we still need the `--save` flag here.